.github/workflows/connectors_tests.yml

```diff
@@ -126,10 +126,12 @@ jobs:
           docker_hub_password: ${{ secrets.DOCKER_HUB_PASSWORD }}
           docker_hub_username: ${{ secrets.DOCKER_HUB_USERNAME }}
           gcp_gsm_credentials: ${{ secrets.GCP_GSM_CREDENTIALS }}
+          gcp_integration_tester_credentials: ${{ secrets.GCLOUD_INTEGRATION_TESTER }}
           sentry_dsn: ${{ secrets.SENTRY_AIRBYTE_CI_DSN }}
           git_branch: ${{ github.head_ref }}
           git_revision: ${{ steps.fetch_last_commit_id_pr.outputs.commit_id }}
           github_token: ${{ env.PAT }}
+          github_maintenance_token: ${{ secrets.GH_PAT_MAINTENANCE_OSS }}
```

Comment:
I would prefer not to have to use this in addition to `env.PAT`, but `env.PAT` isn't scoped properly to run the live tests, and `secrets. GH_PAT_MAINTENANCE_OSS` isn't scoped properly for the rest of the tests.